tests/fixtures/rules/require-class-comment/fixtures.js

```diff
@@ -1,59 +1,68 @@
 module.exports = {
 	valid: {
 		params: '/**\n' +
-		' * Foo\n' +
-		' * @param {string} spec.bar - identifier\n with long description\n' +
-		' * @param {number} spec.baz - number\n' +
-		' */\n' +
-		'model.subclass(function(that, my){\n' +
-		'	my.initialize = function(spec) {\n' +
-		'		my.bar = spec.bar\n' +
-		'		my.baz = spec.baz\n' +
-		'	};\n' +
-		'})',
+			' * Foo\n' +
```

Comment:
Can you please change formatting in a separate commit so we better see what has changed?